cmd/xl-v1-bucket.go

```diff
@@ -65,7 +66,7 @@ func (xl xlObjects) MakeBucketWithLocation(ctx context.Context, bucket, location
 		}, index)
 	}
 
-	writeQuorum := len(storageDisks)/2 + 1
+	writeQuorum := int(math.Ceil(float64(len(storageDisks))/2.0)) + 1
```

Comment:
```suggestion
	writeQuorum := writeQuorum(len(storageDisks))
```

(or just inline below)